### PR TITLE
Convert to ES6

### PR DESCRIPTION
**Pull request by amazingrando**
## What this PR does
Updates the JS to use ES6 syntax. Solves #5.

## How to review
- [ ] Use the gulp tasks and confirm that everything works the same
- [ ] Confirm that running `gulp wpt` fails without a key and works with a key.

### CODE REVIEW

File: gulp-tasks/gulp-pattern-lab.js [hunk 1]
```diff
@@ -5,25 +5,25 @@
...
 
   // Pattern Lab
-  var path = require('path');
-  var yaml = require('js-yaml');
-  var fs = require('fs');
+  const path = require('path');
+  const yaml = require('js-yaml');
+  const fs = require('fs');
 
   module.exports = function (gulp, config, tasks) {
```
Comment (infiniteluke) [inline gulp-tasks/gulp-pattern-lab.js]: `(gulp, config, tasks) => {`

File: gulp-tasks/gulp-pattern-lab.js [hunk 2]
```diff
@@ -5,25 +5,25 @@
...
-    var plPublic = path.join(plRoot, plConfig.publicDir);
-    var plMeta = path.join(plSource, '/_meta');
-    var consolePath = path.join(plRoot, 'core/console');
+    const plRoot = path.join(config.patternLab.configFile, '../..');
+    const plSource = path.join(plRoot, plConfig.sourceDir);
+    const plPublic = path.join(plRoot, plConfig.publicDir);
+    const plMeta = path.join(plSource, '/_meta');
+    const consolePath = path.join(plRoot, 'core/console');
 
     function plBuild(cb) {
```
Comment (infiniteluke) [inline gulp-tasks/gulp-pattern-lab.js]: `const plBuild = (cb) => {`

**Comment by amazingrando** [comment]
@infiniteluke Thanks for the suggestion of using Lebab. I love it! Your requested changes have been made. Thanks!

**Comment by amazingrando** [comment]
@infiniteluke Actually, I posted before I had tested it. There are issues. I'm working to resolve them.

**Comment by amazingrando** [comment]
@infiniteluke Issues resolved. Ready for another review.

**Comment by amazingrando** [comment]
@infiniteluke Thanks for doing the code review! I appreciate it.

**Pull request merged**